devops/roles/analytics-druid/templates/coordinator/runtime.properties

```diff
@@ -0,0 +1,6 @@
+druid.service=druid/coordinator
+druid.host=druid.host={{ groups.coordinator[0] }}
+druid.port=8081
+
+druid.coordinator.startDelay=PT30S
+druid.coordinator.period=PT30S
```

Comment:
The default value for druid.coordinator.period is PT60S. We need to check what will be the optimum value for this property.

Reply:
If a value is not static, please add it into defaults, so that properties files can be keep untouched